src/CommonLib/utility.cxx

```diff
@@ -642,7 +642,7 @@ UTILITY::LogEventF(
 
     STACK_STRU ( strEventMsg, 256 );
 
-    if (SUCCEEDED(strEventMsg.SafeSnwprintf(
+    if (SUCCEEDED(strEventMsg.SafeVsnwprintf(
```

Comment:
Use LogEventF? And why was this changed?

Reply:
ISafeSnwPrintf internally use argslist. If we call SafeSnwprints and pass in va_list, va_list will be treated as one input and values will not be resolved correctly. We need call SafeVsnwprintf which accepts va_list. See our STRU implementation.